statusline/buddy-status.sh

```
@@ -209,6 +316,15 @@ NAME_PAD=$(( ART_CENTER - NAME_LEN / 2 ))
 NAME_LINE="$(printf '%*s%s' "$NAME_PAD" '' "$NAME_WITH_LEVEL")"
 
 DIM=$'\033[2;3m'
+if [ "$COLOR_ENABLED" -eq 0 ]; then
+    C=""
+    NC=""
+    NEUTRAL=""
```

Shellcheck (0.11.0)

[warning] NEUTRAL appears unused. Verify use (or export if used externally).

(SC2034)